src/server/resolvers.ts

```diff
@@ -26,35 +26,33 @@ import Context from './context';
 import { Models } from './models';
 
 function toDietEnum(restriction: string): DietaryRestriction {
-	if (!Object.values(DietaryRestriction).includes(restriction))
+	if (!(restriction in DietaryRestriction))
 		throw new UserInputError(`Invalid dietary restriction ${restriction}`);
 	return restriction as DietaryRestriction;
 }
 
 function toRaceEnum(race: string): Race {
-	if (!Object.values(Race).includes(race)) throw new UserInputError(`Invalid race ${race}`);
```

Comment:
These are not logically equivalent, the new code checks the keys and the old code the values. Not sure if that works or not yet, will look when I get home

Reply:
You're right